test/parallel/test-os.js

```diff
@@ -85,6 +85,12 @@ const hostname = os.hostname();
 is.string(hostname);
 assert.ok(hostname.length > 0);
 
+const DUMMY_PRIORITY = 10
+os.setPriority(DUMMY_PRIORITY)
+const priority = os.getPriority()
+is.number(priority)
+assert.ok(priority === DUMMY_PRIORITY)
```

Comment:
```suggestion
const DUMMY_PRIORITY = 10;
os.setPriority(DUMMY_PRIORITY);
const priority = os.getPriority();
is.number(priority);
assert.strictEqual(priority, DUMMY_PRIORITY);
```
I think the `;`s are necessary to satisfy the linter.